tox.ini

```diff
@@ -3,7 +3,7 @@ envlist = py3
 
 [testenv]
 deps =
-  coverage==4.5.4
+  coverage==5.3
```

Comment:
Coverage version is freezed in setup.py (`install_requires`). Is there a way to de-duplicate it?

Reply:
well.. you are right that actually the freeze here is not required. If the coverage version is frozen in `setup.py`, that suffices. `tox` will automatically install the version which is mentioned in `setup.py`. I only explicitly mentioned it here, just so it stands out. Let me remove it from here.

But we will need to freeze it in one place for sure.  Perhaps in `setup.py`?

Reply:
Removed this from `tox.ini`. CI will pick up the version from `setup.py`.